src/hooks/usePhotoDate.js

```diff
@@ -0,0 +1,14 @@
+const apiKey = "egwuPjIf803mZ9oZJqCH2Gh36n5nEAyqs9TEIXV4";
```

Comment:
It is better practice if this kind of private information we put it  as constant that starts with "REACT_APP_" inside a file name ".env" in our root directory like this:
`REACT_APP_NASA_KEY=[your-secret-code]`
This way we can access this information in our application like this:
`const apiKey = process.env.REACT_APP_NASA_KEY`